src/pages/public_registration/action_accordions/FindRegistration.tsx

```diff
@@ -0,0 +1,106 @@
+
+function isDoi(query: string) {
+  return query.includes('https://doi.org/');
+}
+
+interface FindRegistrationProps {
+  setSelectedRegistration: (registration: Registration | null) => void;
+  selectedRegistration: Registration | null;
+  filteredRegistrationIdentifier: string;
+}
+export const FindRegistration = ({
+  setSelectedRegistration,
+  selectedRegistration,
+  filteredRegistrationIdentifier,
+}: FindRegistrationProps) => {
+  const { t } = useTranslation();
+  const [searchBeforeDebounce, setSearchBeforeDebounce] = useState('');
+  const debouncedSearch = useDebounce(searchBeforeDebounce);
+
+  const fetchQuery: FetchResultsParams = {
+    doi: isDoi(debouncedSearch) ? debouncedSearch : null,
+    query: !isDoi(debouncedSearch) ? debouncedSearch : null,
+  };
+
+  const duplicateRegistrationSearch = useQuery({
+    enabled: debouncedSearch.length > 0,
+    queryKey: ['duplicateRegistration', fetchQuery],
+    queryFn: () => fetchResults(fetchQuery),
+    meta: { errorMessage: t('feedback.error.get_registrations') },
+  });
+
+  const handleSearchAutoCompleteChange = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    setSearchBeforeDebounce(event.target.value);
+  };
+
+  const searchResults = duplicateRegistrationSearch.data?.hits ?? [];
+
+  const searchResultNotContainingToBeDeleted = searchResults.filter(
+    (possibleDuplicate) => possibleDuplicate.identifier !== filteredRegistrationIdentifier
+  );
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+      <Typography variant="h3">{t('unpublish_actions.search_for_duplicate')}</Typography>
+      <Autocomplete
+        options={searchResultNotContainingToBeDeleted}
+        getOptionLabel={(option: Registration | string) => {
+          if (typeof option === 'string') {
+            return option;
+          }
+          return option.entityDescription?.mainTitle ?? '';
+        }}
+        freeSolo
+        loading={duplicateRegistrationSearch.isLoading && debouncedSearch.length > 0}
+        value={selectedRegistration}
+        onChange={(_, newValue) => {
+          if (typeof newValue === 'string') {
+            setSelectedRegistration(null);
+          } else {
+            setSelectedRegistration(newValue);
+          }
+        }}
+        renderInput={(params) => (
+          <TextField
+            {...params}
+            placeholder={t('unpublish_actions.search_duplicate_facets')}
+            variant="filled"
+            label={t('unpublish_actions.duplicate')}
+            onChange={(event) => setSearchBeforeDebounce(event.target.value)}
+            InputProps={{
+              ...params.InputProps,
+              type: 'search',
+              startAdornment: (
+                <IconButton data-testid={dataTestId.startPage.searchButton} title={t('common.search')} size="large">
+                  <SearchIcon />
+                </IconButton>
```

Comment:
denne knappen gjør ikke noe, da søket trigges automatisk basert på input-debounce, så det er kanskje bare ikonet du er interessert i her? Ellers må bruker "tab"-e seg gjennom et element som sier søk, men om man trykker så gjør vi ikke noe søk :p
```suggestion
                <SearchIcon />
```